book/working-with-multiple-records.md

```diff
@@ -0,0 +1,201 @@
+Working With Multiple Records
+=============================
+
+Sometimes one request can create or affect several records in one or more tables
+in such cases its advisable to follow [ACID](https://en.wikipedia.org/wiki/ACID)
+rules. Yii2 supports transactions, isolation levels, and allow you to
+validate data independently, so following the ACID rules is simple.
```

Comment:
i was doubtful to use "properties" since for php it means something different but ok let me change it.